certified-connectors/dox42/apiProperties.json

```diff
@@ -48,7 +48,7 @@
         "templateId": "dynamichosturl",
         "title": "Define dox42 Service URL",
         "parameters": {
-          "x-ms-apimTemplateParameter.urlTemplate": "https://@headers('domainname')/dox42RestService.ashx?@queryParameters('querystring')"
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@headers('domainname')/dox42RestService.ashx?@headers('querystring')"
```

Comment:
Hello @dox42diegoschleis, 

The path template has been changed for the operation, this would fail existing flows, please confirm this will not break the existing flows.

Reply:
I do not understand how existing flows are affected by this change. 

This custom connector is imported as a custom connector on any given environment, so I am assuming that the connector on said environment is a copy of this custom connector and therefore a separate version?

Please help me understand this